tfs/src/main/java/hudson/plugins/tfs/browsers/TeamSystemWebAccessBrowser.java

```diff
@@ -62,13 +62,10 @@ private String getBaseUrlString(ChangeSet changeSet) throws MalformedURLExceptio
     public URL getChangeSetLink(final ChangeSet changeSet) throws IOException {
         final String baseUrlString = getBaseUrlString(changeSet);
         final URL baseUrl = new URL(baseUrlString);

```

Comment:
When did the URL change? Do we need to be backwards compatible?

Reply:
The URL hasn't changed, I believe this URL has always been incorrect. We should be safe.